importer/SeBbrSv.py

```diff
@@ -109,8 +115,9 @@ def set_heritage_bbr(self):
             except ValueError:
                 self.add_to_report("protection_type", bbr_type)
         kulturarv_source = self.create_kulturarv_source(url)
-        self.add_statement(
-            "heritage_status", type_q, qualifier, refs=[kulturarv_source])
+        if type_q:
```

Comment:
Does it happen frequently or is this "just in case"? When/if it happens do we know why?